package.json

```diff
@@ -39,7 +43,7 @@
     "url": "https://github.com/bufferoverflow/verdaccio-gitlab/issues"
   },
   "engines": {
-    "node": ">=6.12.0"
+    "node": ">=8"
```

Comment:
:-) I ignore whether there is a warning on readme, but, worth to mention.

Reply:
It wasn't, I'm adding it now